src/components/App.tsx

```diff
@@ -1,20 +1,32 @@
 import './App.css'
 
-import React, { useState } from 'react'
+import React from 'react'
+import { BrowserRouter, Link, Route, Switch } from 'react-router-dom'
 
-const App: React.FC = () => {
-  const [isOpen, changeIsOpen] = useState<boolean>(false)
+import Home from '@/pages/Home'
+import NewTutorial from '@/pages/NewTutorial'
 
-  return (
+const App: React.FC = () => (
+  <BrowserRouter>
```

Comment:
maybe we should move `BrowserRouter` to `index.js`
```javascript
ReactDOM.render(
  <BrowserRouter>
    <App />
  </BrowserRouter>,
  document.getElementById('main'))
```
most of the examples use this setup